lib/NQuads.js

```diff
@@ -194,18 +194,16 @@ module.exports = class NQuads {
   }
 
   /**
-   * Converts an RDF quad to an N-Quad string (a single quad).
+   * Converts RDF quad components to an N-Quad string (a single quad).
    *
-   * @param quad the RDF quad convert.
+   * @param {Object} s - N-Quad subject component.
+   * @param {Object} p - N-Quad predicate component.
+   * @param {Object} o - N-Quad object component.
+   * @param {Object} g - N-Quad graph component.
    *
-   * @return the N-Quad string.
+   * @return {string} the N-Quad.
    */
-  static serializeQuad(quad) {
-    const s = quad.subject;
-    const p = quad.predicate;
-    const o = quad.object;
-    const g = quad.graph;
-
+  static serializeQuadComponents(s, p, o, g) {
```

Comment:
Could use destructuring here instead: `({subject: s, predicate: p, object: o, graph: g})`. This is internal anyway, so can always change later.